web/src/lightbox.ts

```diff
@@ -532,12 +527,14 @@ export function parse_media_data(media: HTMLElement): Payload {
         user: sender_full_name,
         title: $parent.attr("aria-label") ?? $parent.attr("href"),
         type,
-        preview: util.is_valid_url(preview_src) ? preview_src : "",
+        preview: preview_src && util.is_valid_url(preview_src) ? preview_src : "",
         source: source && util.is_valid_url(source) ? source : "",
         url: url && util.is_valid_url(url) ? url : "",
     };
 
-    asset_map.set(preview_src, payload);
+    if (!$media.hasClass("image-loading-placeholder")) {
+        asset_map.set(canonical_url, payload);
+    }
```

Comment:
Maybe adding a `preview_src &&` wrapping this would be safer, just because it'd be a nasty bug if this condition diverged from the one setting `preview_src = ""` and we started having multiple things share a given cache key.